ci/build.sh

```diff
@@ -18,6 +18,39 @@ function get_branch_to_checkout() {
     || echo "${TRAVIS_BRANCH}"
 }
 
+function bundle_fluentd_plugins() {
+  local VERSION="${1}"
```

Comment:
😍

Reply:
Let's change the VERSION variable into `version` to indicate it's a local one.
```suggestion
  local version="${1}"
```